Core/Core/Network/DownloadManager.swift

```diff
@@ -470,6 +472,60 @@ public class DownloadManager: DownloadManagerProtocol {
             debugLog("SaveFile Error", error.localizedDescription)
         }
     }
+    
+    public func removeAppSupportDirectoryDeprecatedContent() {
+        deleteMD5HashedFolders()
```

Comment:
How about doing the work directly here that you are doing in `deleteMD5HashedFolders` function?

Reply:
For our specific scenario, the suggested name `removeAppSupportDirectoryUnusedContent` seems fitting. However, the name `deleteMD5HashedFolders` explicitly describes the deletion of MD5 hashed folders (not any other content). If we integrate the implementation of `deleteMD5HashedFolders()` directly into `removeAppSupportDirectoryUnusedContent()`, it might affect the clarity of the abstraction level. Do you have any suggestions or thoughts on finding a more suitable name?